src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj

```diff
@@ -32,6 +37,13 @@
     <Compile Include="System\Runtime\InteropServices\ComTypes\tymed.cs" />
   </ItemGroup>
   <ItemGroup>
+    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' != 'netcore50aot'" />
```

Comment:
There's three different conditions here, it will be quite messy to have different ItemGroups for these, IMO. How would you organize them?